packages/react-router/lib/components.tsx

```diff
@@ -217,6 +218,10 @@ export function Navigate({
     if (dataRouterState && dataRouterState.navigation.state !== "idle") {
       return;
     }
```

Comment:
This could probably be removed after navigateCalledRef was added.
No tests fails with this statement removed.